conda/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM continuumio/miniconda3
+FROM continuumio/miniconda3:4.7.2
```

Comment:
@henryiii Why are you setting a specific version?

```suggestion
FROM continuumio/miniconda3
```
If there is a reason I guess you meant:
```suggestion
FROM continuumio/miniconda3:4.7.12
```

Reply:
I generally like setting versions in docker files. It make the build reproducible and keeps docker caching from rebuilding when the base changes. If the version is really not a good idea, it can be dropped, though.

Reply:
Thanks for catching my mistake!